src/main/webapp/app/shared/model/team.model.ts

```diff
@@ -35,5 +37,6 @@ export class Team implements ITeam {
         public daysUntilExpiration?: number
     ) {
         this.pureTrainingTeam = this.pureTrainingTeam || false;
+        this.official = this.official || false;
```

Comment:
I suppose you meant this instead:
```suggestion
        this.official = official || false;
```